6_transformer_translation.ipynb

```diff
@@ -0,0 +1,1266 @@
+{
```

Comment:
<p><code>tgt</code> seems more target than <code>trg</code> to me, but doesn't matter</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scoutbee/pytorch-nlp-notebooks/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>